guides/doc-Planning_Guide/topics/Org_Loc_and_Life_Cycle_Environments.adoc

```diff
@@ -1,5 +1,10 @@
+ifdef::katello[]
 [[chap-Architecture_Guide-Org_Loc_and_Life_Cycle_Environments]]
 == Organizations, Locations, and Life Cycle Environments
+endif::[]
+ifndef::katello[]
+== Organizations and Locations
```

Comment:
```suggestion
 [[chap-Architecture_Guide-Org_Loc]
== Organizations and Locations
```
Something as simple as this would mean you've a unique anchor and can build, link to, and reference this easily.